test/models/game_test.rb

```diff
@@ -1,7 +1,33 @@
 require 'test_helper'
 
 class GameTest < ActiveSupport::TestCase
-  # test "the truth" do
-  #   assert true
-  # end
+  def setup
+    @g = Game.create(name: "New Game", white_player_id: 1, black_player_id: 1)
```

Comment:
Thanks -- I was following Ken's example in the gist (ie. when he was testing via rails console), and didn't think about how that would change things.  I've changed it.